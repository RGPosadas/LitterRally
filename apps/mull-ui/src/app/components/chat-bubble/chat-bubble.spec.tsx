import { render } from '@testing-library/react';
import React from 'react';
import renderer from 'react-test-renderer';
import ChatBubble from './chat-bubble';

describe('ChatBubble', () => {
  it('should render successfully', () => {
    const { baseElement } = render(<ChatBubble />);

    expect(baseElement).toBeTruthy();
  });

  it('should match snapshot', () => {
    const tree = renderer.create(<ChatBubble />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
