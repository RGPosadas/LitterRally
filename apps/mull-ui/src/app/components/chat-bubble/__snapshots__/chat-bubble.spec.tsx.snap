// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ChatBubble should match snapshot 1`] = `
<div
  className="chat-container"
>
  <p
    className="announcement-time"
  >
    
  </p>
  <div
    className="other-user-chat-container"
  >
    <img
      alt="user"
      className="user-picture"
      onClick={null}
      src=""
    />
    <p
      className="chat-bubble"
    >
      
    </p>
  </div>
</div>
`;
